### PR TITLE
DOC: link np.interp to SciPy's interpolation functions (closes #14154)

### LINKED ISSUE #14154: DOC: link to SciPy's multidimensional interpolation functions
### Reproducing code example:

np.interp currently requires its `fp` argument to be one-dimensional.  An obvious generalization would be to allow interpolation of `R->R^n` functions, by allowing `fp` to have shape (..., len(x)) (or (len(x), ...) -- whichever is consistent with e.g. polyfit).  Obviously the shapes of `left` and `right` should be adjusted accordingly.

Thoughts?

### Numpy/Python version information:

```
1.17.0 3.7.3 (default, Mar 27 2019, 22:11:17) 
[GCC 7.3.0]
```

### PR DESCRIPTION
**Pull request by lschroefl**
I've added a cross-link to SciPy's interpolation functions in the docstring of np.interp as suggested in #14154

### CODE REVIEW

File: numpy/lib/function_base.py [hunk 1]
```diff
@@ -1332,6 +1332,10 @@ def interp(x, xp, fp, left=None, right=None, period=None):
         If `xp` or `fp` are not 1-D sequences
         If `period == 0`
 
+    See also
```
Comment (rossbar) [inline numpy/lib/function_base.py:1335]:
```suggestion
    See Also
```
per the [numpydoc standard](https://numpydoc.readthedocs.io/en/latest/format.html#docstring-standard).

**Review by rossbar — CHANGES_REQUESTED** [review]
Just a minor formatting change with the "See also" heading title, but otherwise LGTM.

IMO this is sufficient to address #14154.

**Review by rossbar — APPROVED** [review]
LGTM, thanks @LSchroefl

**Comment by rossbar** [comment]
Close/Reopen for CI

**Comment by mattip** [comment]
Build failure unrelated. Thanks @LSchroefl. @rossbar - feel free to push the green button to merge PRs that you feel comfortable with.

**Pull request merged**